core/src/test/java/org/apache/calcite/rex/RexProgramTest.java

```diff
@@ -1774,6 +1770,27 @@ private void checkExponentialCnf(int n) {
     checkSimplify2(e, "SEARCH(?0.int0, Sarg[10])", "=(?0.int0, 10)");
   }
 
+  @Test void testSimplifyInAnd() {
+    // deptno in (20, 10) and deptno = 10
+    //   ==>
+    // deptno = 10
+    checkSimplify2(
+        and(
+            in(vInt(), literal(20), literal(10)),
+            eq(vInt(), literal(10))),
+        "SEARCH(?0.int0, Sarg[10])",
+        "=(?0.int0, 10)");
+
+    // deptno in (20, 10) and deptno = 30
+    //   ==>
+    // false
+    checkSimplify(
+        and(
+        in(vInt(), literal(20), literal(10)),
+        eq(vInt(), literal(30))),
+        "false");
+  }
+
```

Comment:
I find `deptno > 0 and deptno in (20, 10)` can' t be simplified as `deptno > 0`

Reply:
Thanks, i have updated the test case.